pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -525,25 +525,87 @@ LogicalResult ConvertAtenOp<AtenTanhOp>::matchAndRewrite(
   }
 }
 
+// Convert a Aten::Relu to HLO
+// Relu(x) = 0 if x < 0 else x
+template <>
+LogicalResult ConvertAtenOp<AtenReluOp>::matchAndRewrite(
+    AtenReluOp op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  Location loc = op.getLoc();
+  Value input = adaptor.self();
+  auto inputTy = input.getType().cast<TensorType>();
+  Value zero = chlo::getConstantLike(rewriter, loc, 0.0, input);
+  Value compareGtZero = rewriter.create<mhlo::CompareOp>(
+      loc, input, zero, mhlo::ComparisonDirection::GT);
+  rewriter.replaceOpWithNewOp<mhlo::SelectOp>(
+      op, inputTy, compareGtZero, input, zero);
+  return success();
+}
+
+// Convert a Aten::LeakyRelu to HLO
+// LeakyRelu(x) = max(0, x) + negative_slop * min(0, x)
+template <>
+LogicalResult ConvertAtenOp<AtenLeakyReluOp>::matchAndRewrite(
```

Comment:
nit: FileCheck based UT or open an issue to track this TODO?.

Reply:
Thanks, I created an issue https://github.com/alibaba/BladeDISC/issues/385 to trace this task.